app/assets/src/components/Samples.jsx

```diff
@@ -566,15 +569,11 @@ class Samples extends React.Component {
       let derivedOutput = sample.derived_sample_output;
       let runInfo = sample.run_info;
       let uploader = sample.uploader.name;
-      let descrip = runInfo.job_status_description;
-      let statusClass = !descrip
-        ? this.applyChunkStatusClass(runInfo)
-        : this.applyClass(descrip);
-      let status = !descrip ? this.getChunkedStage(runInfo) : descrip;
+      let status = runInfo.result_status_description;
 
       const stageStatus = (
-        <div className={`${statusClass} status`}>
-          {this.appendStatusIcon(status)}
+        <div className={`${this.statusClass(status)} status`}>
```

Comment:
maybe statusClass() and statusIcon() could also be combined if this is the only placed they're used